docs/_running-tests/automation.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: page
+title: Running Tests in Automation
+---
+In simple cases individual tests can be run by simply loading the page
+in a browser window. More detail on that approach is available in [Running
+Tests][running].
+
+For running larger groups of tests, or running tests frequently, this is not a
+practical approach and several better options exist.
+
+## From the Command Line
+
+The simplest way to run tests is to use the `wpt run` command from the
+root of the repository. This will automatically load the tests in the
+chosen browser, and extract the test results. For example to run the
+`dom/historical.html` tests in a local copy of Chrome:
+
+    ./wpt run chrome dom/historical.html
+
+Or to run in a specified copy of Firefox:
+
+    ./wpt run --binary ~/local/firefox/firefox firefox dom/historical.html
+
+`./wpt run --help` lists the supported products.
```

Comment:
This is also in the next paragraph.

Reply:
Yup. Doesn't hurt to fix that in this patch